joomla/joomla-cms
# [4.0] [com_languages] Parameterized queries

### Summary of Changes

Adds parameterized queries and cleans up SQL in `com_languages`.

### Testing Instructions

Set up a multilingual site. Use `com_languages` in all sorts of way, e.g.:

Search, install languages.
Create/modify content languages.
Create language overrides.
Check multilanguage status.

### Expected result

Works like before.

### Documentation Changes Required

No.
